index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles/style.css">
+  <link rel="stylesheet" href="styles/index-style.css">
+
+  <title>Fansite</title>
+
+</head>
+
+<body class="container">
+  <header >
+    <h1>  Guy Fieri Fansite </h1>
+  </header>
+
+  <nav>
+    <ul>
+      <img id="guy-gif"alt="Guy Gif" src="https://boldtv.com/wp-content/uploads/2020/10/guy-fieri-chefs-kiss.gif">   
+      <li>Home</li>
+      <li><a href="pages/facts.html">Facts</a></li>
+      <li><a href="pages/gallery.html">Gallery</a></li>  
+      <p></p>
+      <img id="guy-gif"alt="Guy Gif" src="https://boldtv.com/wp-content/uploads/2020/10/guy-fieri-chefs-kiss.gif">   
+    </ul>  
+  </nav>
+      
+  <main>   
+      <h2>✨🦐🍽️Welcome to our fansite!🍽️🍖✨</h2>
+      <h3> "We're takin' you on a road rockin' trip down to Flavortown, where the gravitational force of bacon warps the laws of space and time." </h3>
+
+      <section>
+        <img id="gif"alt="Guy Gif" src="https://i.giphy.com/media/siavMVEdnhd5Q6QScP/giphy.webp">
+        <img id="gif2"alt="Guy Gif" src="https://hips.hearstapps.com/esquire/assets/16/04/1454080670-200-4.gif"> 
+        <img id="gif3"alt="Guy Gif" src="https://i.giphy.com/media/siavMVEdnhd5Q6QScP/giphy.webp"> 
+      </section>
+  </main>
```

Comment:
Some indentation is 2-spaces and some is 4-spaces, to make things easy to read and reduce possible confusion, we should pick one and be consistent across the project.